up.sh

```diff
@@ -40,14 +33,10 @@ echo -n > .bootstrap/bootstrapped_deployments
 if [[ "$*" == *tracing* ]]
 then
   set_docker_environment_value "TRACING_ENABLED" "true"
-else
-  set_docker_environment_value "TRACING_ENABLED" "false"
 fi
 if [[ "$*" == *instrumentation* ]]
 then
   set_docker_environment_value "INSTRUMENTATION_ENABLED" "1"
-else
```

Comment:
See comment for tracing. Same applies here

Reply:
replied there